nimble-storage/nimble-csp-api-swagger-v2.json

```diff
@@ -1143,6 +1193,32 @@
         }
       }
     },
+    "ContainerReplicationPartner": {
+      "type": "object",
+      "properties": {
+        "alive": {
+          "type": "boolean"
+        },
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "repl_direction": {
+          "type": "string",
+          "enum": [
+            "upstream",
```

Comment:
Minor comment...  The values "upstream" and "downstream" don't seem right for a "replication direction".  Replication always goes from upstream to downstream.  Consider calling it "repl_role" or "repl_position" instead of "repl_direction".

Also change "bi_directional" to "bidirectional".

Reply:
These are popping in from some of the existing NimOS definitions.